pkg/pot/ast.go

```diff
@@ -46,6 +46,9 @@ func TokensToPo(tokens []Token) (Po, error) {
 	header := headerNode{}
 
 	for _, token := range tokens {
+		token.Value = strings.ReplaceAll(token.Value, "\\n ", "\n")
+		token.Value = strings.ReplaceAll(token.Value, "\\n", "\n")
```

Comment:
```suggestion
replacer := strings.NewReplacer("\\n ", "\n", "\\n", "\n")
token.Value = replacer.Replace(token.Value)
```
Same thing, but intend should be clearer